### PR TITLE
feat: add roles

### PR DESCRIPTION
**Pull request by cyx**
## Description

This PR adds list, show, create, update, delete commands for the `role` resource.

A quick demo is shown below showcasing all the commands.

[asciicast](https://asciinema.org/a/Q2WDzn5LZQNgofaxyEHkMyJJ9)

### CODE REVIEW

File: internal/display/roles.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+func (v *roleView) AsTableHeader() []string {
+	return []string{"Name", "Role ID", "Description"}
+}
+
+func (v *roleView) AsTableRow() []string {
+	return []string{
+		v.Name,
+		ansi.Faint(v.ID),
```
Comment (Widcket) [inline internal/display/roles.go:21]: Can we have the ID go first? Like in the other list commands.

File: internal/cli/roles.go [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+	return cmd
+}
+
+func listRolesCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Args:    cobra.NoArgs,
+		Short:   "List your roles",
+		Long: `List your existing roles. To create one try:
```
Comment (Widcket) [inline internal/cli/roles.go:61]: Can you please run the file through `gofmt`?

**Review by Widcket — CHANGES_REQUESTED** [review]
LGTM, just a couple of tiny details.

File: internal/display/roles.go (hunk 1 above)
Comment (cyx) [reply internal/display/roles.go:21]: Yep, done

File: internal/cli/roles.go (hunk 2 above)
Comment (cyx) [reply internal/cli/roles.go:61]: Yeah, I think this is special gofmt logic.

**Review by Widcket — APPROVED** [review]

**Pull request merged**